docs/docs/interactions.md

```diff
@@ -4,31 +4,31 @@ title: Cross handler interactions
 sidebar_label: Cross handler interactions
 ---
 
-Gesture handlers can "communicate" with each other in order to allow for defining complex gesture interactions and to control how they [activate](state.md#active) in certain scenarios.
+Gesture handlers can "communicate" with each other to support complex gestures and control how they *[activate](state.md#active)* in certain scenarios.
 
-There are two means of such a control at the moment described in the sections below.
-In each of the cases it is necessary to provide a reference of one handler as a property to the other.
+There are two means of momentary gesture control described in the sections below.
```

Comment:
This changes meaning – this sentence was mentioning that we have two ways of making "complex" gestures right now. I think this can be rewritten to:

"There are two means of achieving that described in the sections below."

Reply:
ahh okay